pkg/monitor/monitor/util.go

```diff
@@ -320,9 +320,6 @@ chmod 777 /data/prometheus /data/grafana
 			},
 		},
 		Command: command,
-		SecurityContext: &core.SecurityContext{
-			RunAsUser: pointer.Int64Ptr(0),
-		},
```

Comment:
in OpenShift, `0` is not allowed by default. the root user is not required here, we can omit it and use the default value in the cluster.